lessons/githubPages/lesson.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: page
+title: 'Web design with Jekyll/Github Pages'
+visible: false
+---
+<!-- change visible to true if you want it on the site -->
+
+ - **Authors**: Dawn Walker [@dcwalk](https://github.com/dcwalk)
```

Comment:
You're right. Looked into it: https://help.github.com/articles/mentions-on-github-pages/